bluebrain/repo-bluebrain/packages/py-bluepy/package.py

```diff
@@ -13,14 +13,14 @@ class PyBluepy(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/bluepy.git"
 
     version("develop", branch="main")
-    version("2.5.0", tag="bluepy-v2.5.0")
+    version("2.5.1", tag="bluepy-v2.5.1")
     version("2.4.5", tag="bluepy-v2.4.5")
 
     depends_on("py-setuptools", type=("build", "run"))
 
     depends_on("py-libsonata@0.1.7:0", type=("build", "run"))
     depends_on("py-pandas@1", type=("build", "run"))
-    depends_on("py-bluepy-configfile@0.1.15:0", type=("build", "run"))
+    depends_on("py-bluepy-configfile@0.1.20:0,develop", type=("build", "run"))
```

Comment:
I would feel more comfortable with this being open-ended, to allow for other non-numerical versions.

Reply:
Yeah ok, since we control both bluepy and bluepy-configfile it should be safe 👍